### PR TITLE
APR-1270 schema changes to support RoATP assessments

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Database/Tables/Apply.sql [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
     [CreatedAt] DATETIME2 NOT NULL, 
     [CreatedBy] NVARCHAR(256) NOT NULL, 
     [UpdatedAt] DATETIME2 NULL, 
     [UpdatedBy] NVARCHAR(256) NULL, 
     [DeletedAt] DATETIME2 NULL, 
-    [DeletedBy] NVARCHAR(256) NULL
+    [DeletedBy] NVARCHAR(256) NULL,
+    [FinancialReviewStatus] NVARCHAR(20) NOT NULL DEFAULT 'Draft',
+    [GatewayReviewStatus] NVARCHAR(20) NOT NULL DEFAULT 'Draft',
+    [FinancialGrade] NVARCHAR(MAX) NULL
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Database/Tables/Apply.sql]: Could we shove these columns up in the table so its next to AssessorReviewStatus? Would just make a a bit more readable? Was thinking in this order:   GatewayReviewStatus , AssessorReviewStatus , FinancialReviewStatus , FinancialGrade

File: src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.refactorlog [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.refactorlog]: Is this file needed?

File: src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.sqlproj [hunk 3]
```diff
@@ -89,4 +89,7 @@
   <ItemGroup>
     <None Include="SFA.DAS.ApplyService.Database.publish.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <RefactorLog Include="SFA.DAS.ApplyService.Database.refactorlog" />
```
Comment (martin-hornsby) [inline src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.sqlproj]: Is this needed?

Comment (David-B-Read) [reply src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.sqlproj]: Removed

File: src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.refactorlog (hunk 2 above)
Comment (David-B-Read) [reply src/SFA.DAS.ApplyService.Database/SFA.DAS.ApplyService.Database.refactorlog]: Removed

File: src/SFA.DAS.ApplyService.Database/Tables/Apply.sql (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.ApplyService.Database/Tables/Apply.sql]: Done
